.travis.yml

```diff
@@ -18,22 +19,14 @@ matrix:
       - coverage run -a ./gff3tool/bin/gff3_merge.py -g1 example_file/new_models_w_replace.gff3 -g2 example_file/reference.gff3 -f example_file/reference.fa -og merged.gff -r merged_report.txt -noAuto
       - coverage run -a ./gff3tool/bin/gff3_sort.py -g example_file/example.gff3 -og example-sorted.gff3
       - coverage run -a ./gff3tool/bin/gff3_to_fasta.py -g example_file/example.gff3 -f example_file/reference.fa -st all -d simple -o test_sequences
-
-  - os: linux
-    language: python
-    python: 2.7
-    install:
-      - pip install --no-cache-dir gff3tool
```

Comment:
This should not be removed. It's for testing installation package directly from PyPI.